pkgs/development/haskell-modules/patches/hevm-update-deps.patch

```diff
@@ -0,0 +1,129 @@
+diff --git a/hevm.cabal b/hevm.cabal
+index cf36961e..3b31f595 100644
+--- a/hevm.cabal
++++ b/hevm.cabal
+@@ -71,6 +71,10 @@ library
+     -Wall -Wno-deprecations
+   extra-libraries:
+     secp256k1, ff
++  if os(darwin)
++     extra-libraries: c++
++  else
++     extra-libraries: stdc++
+   c-sources:
+     ethjet/tinykeccak.c, ethjet/ethjet.c
+   cxx-sources:
+@@ -88,7 +92,7 @@ library
+     transformers                      >= 0.5.6 && < 0.6,
+     tree-view                         >= 0.5 && < 0.6,
+     abstract-par                      >= 0.3.3 && < 0.4,
+-    aeson                             >= 1.5.6 && < 1.6,
```

Comment:
4. Instead of patching these out, you could get away with just using `doJailbreak`.  That should require less maintenance in the long-run (but be slightly harder to debug when it ends up no longer working).